sdk/keyvault/azure-security-keyvault-administration/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Release History
 
-## 4.1.0-beta.1 (Unreleased)
+## 4.0.2 (2021-08-12)
+
+### Other Changes
```

Comment:
Delete this empty section

Reply:
It's not empty, it's a parent header of `Dependency Updates`.